src/main/java/frc/robot/configs/Season2024RobotConfig.java

```diff
@@ -0,0 +1,394 @@
+package frc.robot.configs;
+
+import edu.wpi.first.math.geometry.Rotation3d;
+import edu.wpi.first.math.geometry.Transform3d;
+import edu.wpi.first.math.geometry.Translation3d;
+import edu.wpi.first.math.util.Units;
+import frc.lib.team3061.RobotConfig;
+import frc.lib.team3061.drivetrain.swerve.SwerveConstants.SwerveType;
+
+/*
+ * Refer to the README for how to represent your robot's configuration. For more information on
+ * these methods, refer to the documentation in the RobotConfig class.
+ */
+public class Season2024RobotConfig extends RobotConfig {
```

Comment:
Let's delete this for now and create it when the robot is more designed. If you want to keep it, copy the PracticeBotConfig.java after it is updated based on my comments.